evals/eval_swe_bench.py

```
@@ -0,0 +1,128 @@
+    assert test_data[0]["instance_id"] == test_data_preprocessed[0]["instance_id"]
+    filepath = Path("SWE-bench_testsample")
+    if filepath.exists():
+        from datasets import Dataset
+        dataset = Dataset.load_from_disk(filepath)
+    else:
+        dataset = download_instances(test_data, filepath)
+    
+    cognee_preds, cognee_preds_dataset = await get_preds(dataset, with_cognee=True)
+    # nocognee_preds = await get_preds(dataset, with_cognee=False)
+    test_specs = list(map(make_test_spec, test_data))
+    results = await evaluate(test_specs, cognee_preds)
```

Ruff

Local variable `results` is assigned to but never used

Remove assignment to unused variable `results`

(F841)